extension/src/experiments/workspace.ts

```diff
@@ -429,6 +440,36 @@ export class WorkspaceExperiments extends BaseWorkspaceWebviews<
     )
   }
 
+  private async checkOrAddPipeline(cwd: string) {
+    const stages = await this.internalCommands.executeCommand(
+      AvailableCommands.STAGE_LIST,
+      cwd
+    )
+
+    if (!stages) {
+      const selectValue = 'select'
+      const pathOrSelect = await quickPickOneOrInput(
+        [{ label: 'Select from file explorer', value: selectValue }],
+        {
+          defaultValue: '',
+          placeholder: 'Path to script',
```

Comment:
We should probably be more clear that only absolute paths work, not relative. Personally, I think that we should allow relative paths in the input if possible. Seems easier to type out 🤔